pkg/operator/staticpod/controllers.go

```diff
@@ -104,7 +104,7 @@ type Builder interface {
 	// the installer pod is created for a revision.
 	WithCustomInstaller(command []string, installerPodMutationFunc installer.InstallerPodMutationFunc) Builder
 	WithPruning(command []string, staticPodPrefix string) Builder
-	WithPodDisruptionBudgetGuard(operatorNamespace, operatorName, readyzPort, readyzEndpoint string, createConditionalFunc func() (bool, bool, error)) Builder
+	WithPodDisruptionBudgetGuard(operatorNamespace string, operatorName string, readyZConf monitor.ReadyZConf, createConditionalFunc func() (bool, bool, error)) Builder
```

Comment:
readyZConf should be passed by pointer to avoid unnecessary copy of the struct.

Reply:
good point, will fix it along with anything else the 2nd reviewer needs